packages/credit-protocol/index.ts

```diff
@@ -418,15 +418,31 @@ export default class CreditProtocol {
     return config.gasPrice
   }
 
-  async getTxCost(currency: string, gasNeeded: number) {
+  async getTransactionCosts(currency: string, gasNeeded: number) {
     try {
       const gasPrice = await this.getGasPrice()
       const rate = await this.getEthExchange(currency)
+      const weiCost = gasPrice * gasNeeded
+      const ethCost = weiCost / WEI_PER_ETH
+      const currencyCost = Math.max( 0.01, ethCost * Number(rate) )
+
+      return {
+        ethCost,
+        ethCostFormatted: `${ethCost}`.slice(0,6),
+        currencyCost,
+        currencyCostFormatted: `${currencySymbols(currency)}${formatSettlementCurrencyAmount(String(currencyCost), false)}`,
```

Comment:
It's not entirely clear what this line is doing: why the `String` cast is necessary, or what the `false` means. Might there be another way to write this such that its intent becomes more clear?

Reply:
Most (all?) of the `format` functions require String parameters.
The second function parameter is `isEth`, which is clearly `false` in this case.